tools/regenerateAll.bat

```diff
@@ -0,0 +1,72 @@
+@echo off
+set repoUser=olydis
+set repoBranch=batches
+set version=latest
+call ..\src\SDKs\StreamAnalytics\Management.StreamAnalytics\generate.cmd                              %version% %repoUser% %repoBranch% noInstall
+call ..\src\SDKs\Storage\Management.Storage\generate.cmd                                              %version% %repoUser% %repoBranch% noInstall
+:: ...
+call ..\src\SDKs\SqlManagement\Management.Sql\generate.cmd                                            %version% %repoUser% %repoBranch% noInstall
+call ..\src\SDKs\ServiceFabric\Management.ServiceFabric\generate.cmd                                  %version% %repoUser% %repoBranch% noInstall
+:: ...
+call ..\src\SDKs\ServerManagement\Management.ServerManagement\generate.cmd                            %version% %repoUser% %repoBranch% noInstall
+:: ...
+call ..\src\SDKs\Resource\Management.ResourceManager\generate.cmd                                     %version% %repoUser% %repoBranch% noInstall
+call ..\src\SDKs\RedisCache\Management.Redis\generate.cmd                                             %version% %repoUser% %repoBranch% noInstall
+:: ?
+call ..\src\SDKs\RecoveryServices\Management.RecoveryServices\generate.cmd                            %version% %repoUser% %repoBranch% noInstall
+:: renames, version, ?
+call ..\src\SDKs\RecoveryServices.SiteRecovery\Management.RecoveryServices.SiteRecovery\generate.cmd  %version% %repoUser% %repoBranch% noInstall
+call ..\src\SDKs\RecoveryServices.Backup\Management.RecoveryServices.Backup\generate.cmd              %version% %repoUser% %repoBranch% noInstall
+:: ?
+call ..\src\SDKs\PowerBIEmbedded\Management.PowerBIEmbedded\generate.cmd                              %version% %repoUser% %repoBranch% noInstall
+:: stuff missing

```

Comment:
well, *I* was actually wondering whether you may have use for the script. It makes no sense in the specs repo, I thought maybe you have value for such a script (automated tests or similar) - will remove!

---
In reply to: [129956036](https://github.com/Azure/azure-sdk-for-net/pull/3489#discussion_r129956036) [](ancestors = 129956036)